packages/block-library/src/navigation-link/index.php

```diff
@@ -218,7 +220,7 @@ function render_block_core_navigation_link( $attributes, $content, $block ) {
 
 	if ( isset( $block->context['showSubmenuIcon'] ) && $block->context['showSubmenuIcon'] && $has_submenu ) {
 		// The submenu icon can be hidden by a CSS rule on the Navigation Block.
-		$html .= '<span class="wp-block-navigation__submenu-icon">' . block_core_navigation_link_render_submenu_icon() . '</span>';
+		$html .= '<span class="wp-block-navigation__submenu-icon">' . esc_html( block_core_navigation_link_render_submenu_icon() ) . '</span>';
```

Comment:
Doesn‘t the icon contain html? If so, this is incorrect

Reply:
Yes, the icon is an inline SVG and this change is making it display as text: 
<img width="375" alt="Screen Shot 2022-01-13 at 1 04 33 pm" src="https://user-images.githubusercontent.com/8096000/149252845-861c5b22-c3fc-455d-a02a-03f8cec68543.png">